experimental/dds/tree/src/id-compressor/AppendOnlySortedMap.ts

```diff
@@ -40,6 +40,20 @@ export class AppendOnlySortedMap<K, V> {
 		return this.elements[this.elements.length - 2] as K;
 	}
 
+	/**
+	 * @returns the min value in the map.
+	 */
+	public minValue(): V | undefined {
+		return this.elements[1] as V;
```

Comment:
`as V | undefined`

Reply:
likewise for similar patterns in this file

Reply:
Good call.